### DISCUSSION TITLE
Help with interpreting PAE plots

### DISCUSSION THREAD

**Discussion by jsquaredosquared** [KosinskiLab/AlphaPulldown#400] [Q&A] [answered]
Sorry if this is a silly question, but does the direction of the diagonal in the PAE plot matter? After running AlphaPulldownSnakemake, all the PAE plots have a diagonal going from the bottom left to the top right, whereas in all the examples I've seen the diagonal goes from the top left to the bottom right.

Also, the PAE plots generated do not have black lines delineating where one protein ends and the other begins.

Here is one example after I ran AlphaPulldownSnakemake:
![image](https://github.com/user-attachments/assets/3abc951b-85c3-4799-9f92-7eba5e890ed7)

Compare this to the one in AlphaPulldown example 1:
![image](https://github.com/user-attachments/assets/27c73487-3820-455e-b5cd-1349a497bdc3)

I would appreciate any help understanding the causes of these differences.

Thanks 🙂

**Comment by DimaMolod** [comment, marked as answer]
Thank you for pointing this out. In the new plots, the origin of the plot seems to be in the bottom-left corner, whereas it used to be in the upper-left corner. I mark this as a bug and we will fix this asap.

**Comment by DimaMolod** [comment]
@jsquaredosquared, could you try it again, please?

**Comment by jsquaredosquared** [comment]
Thanks for the response :D

The HPC cluster I used is undergoing maintenance, so I may not be able to try again until next week. I will let you know once I have tried it.

**Comment by jsquaredosquared** [comment]
Hi @DimaMolod, looks like the orientation is now correct. However, when there is more than one protein, not all the black lines are drawn to separate them, even though the tick marks are visible on the axes.

![Q6BDS2_and_Q6BDS2_and_Q96T88_PAE_plot_ranked_0](https://github.com/user-attachments/assets/fa81e4e5-818a-43cb-b401-12f5d74180ac)

**Reply by DimaMolod** [reply]
Thank you, @jsquaredosquared, for thorough testing! Should be fixed now.
